eng/Versions.props

```diff
@@ -6,6 +6,7 @@
     <AssemblyVersion>15.1.0.0</AssemblyVersion>
     <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
     <DotNetUseShippingVersions>true</DotNetUseShippingVersions>
+    <DotNetFinalVersionKind>release</DotNetFinalVersionKind>
```

Comment:
Put this line on the same line as the 16.11 VersionPrefix definition, so it causes a merge conflict with 17.0 (otherwise we're likely to accidentally have release branding in main for a bit).